tests/SubjectsProviderTest.php

```diff
@@ -107,11 +107,15 @@ public function testSearchWithUnsupportedQuery(): void
             throw new \Exception($errstr, $errno);
         }, E_ALL);
 
-        $this->expectException(\Exception::class);
-        $this->expectExceptionMessage('Unknown option keys: unknown');
-        $response = $provider->search($querySearch + ['unknown' => 'unknown']);
-
-        $this->assertEquals(['data' => 'test@fakturoid.cz'], $response->getBody(true));
+        try {
+            $this->expectException(\Exception::class);
+            $this->expectExceptionMessage('Unknown option keys: unknown');
+            $response = $provider->search($querySearch + ['unknown' => 'unknown']);
+
+            $this->assertEquals(['data' => 'test@fakturoid.cz'], $response->getBody(true));
+        } finally {
+            restore_error_handler(); // Odstranění chybového handleru
```

Comment:
Could you re-phrase it in english? Though it might just be _Restore error handler_ in which case we can delete it anyway. :)